config/config_test.go

```diff
@@ -1202,6 +1202,7 @@ func getExpectedDefaultConfig() Configuration {
 				DefaultTTL: utils.CASSANDRA_DEFAULT_TTL_SECONDS,
 			},
 			Redis: Redis{
+				Hosts:             []string{},
```

Comment:
I don't think we need this. The test is able to pass without it.

Reply:
I beat a lot to find what was wrong before I explicit declare this here.
Without, it fail because the array is nil.